common-theurgy.lic

```diff
@@ -142,12 +142,30 @@ module DRCTH
     DRCI.put_away_item?(DRC.left_hand, container)
   end
 
+  def sprinkle_holy_water?(theurgy_supply_container, water_holder, target)
+    did_sprinkle_holy_water = (
+      DRCI.get_item?(water_holder, theurgy_supply_container) &&
+      sprinkle?(water_holder, target) &&
+      DRCI.put_away_item?(water_holder, theurgy_supply_container)
+    (
```

Comment:
🧹  need closing paren
```suggestion
    )
```

Reply:
This looks to be doing exactly the same as the existing `sprinkle_holy_water` method but just returns a `true` if all three actions are true, else `false`.

Is it necessary to have the new method or simply add the `&&` to the existing methods?

Reply:
I don't intend to keep both. I'll clean up when we have these, and remove the non-booleans.